lib/rules/no-descending-specificity/index.mjs

```diff
@@ -16,16 +16,16 @@ import validateOptions from '../../utils/validateOptions.mjs';
 const ruleName = 'no-descending-specificity';
 
 const messages = ruleMessages(ruleName, {
-	expected: (b, a) => {
-		return `Expected selector "${b}" to come before selector "${a}"`;
+	expected: (b, a, line) => {
```

Comment:
You have to update

https://github.com/stylelint/stylelint/blob/7cfc5e5bd64379fa1f7ceca7c8e9e6601abada2d/types/stylelint/index.d.ts#L770-L774

Reply:
Done, thanks